x-pack/plugin/core/template-resources/src/main/resources/ecs@mappings.json

```diff
@@ -60,6 +60,87 @@
             "match_mapping_type": "string"
           }
         },
+        {
+          "ecs_explicit_keywords": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "path_match": [
+              "*registry.data.bytes"
+            ],
+            "match_mapping_type": "string"
+          }
+        },
```

Comment:
This allows me to add `*.bytes` to the long-mapping dynamic template. Otherwise I would have to add:
```
"*.ingress.bytes",
"*.egress.bytes",
"*.write.bytes",
"*.read.bytes",
"*.body.bytes",
"*server.bytes",
"*client.bytes",
"*http.request.bytes",
"*http.response.bytes",
"*network.bytes",
"*destination.bytes",
"*source.bytes",
```